devkit/manager/src/fund-via-wallet.ts

```diff
@@ -0,0 +1,93 @@
+import { Wallet, Contract, providers, BigNumber, utils } from "ethers";
+import { createLogger } from "./utils";
+
+const log = createLogger(["fund-via-wallet"]);
+
+/**
+ * Fund a recipient using a wallet.
+ * @param privateKey
+ * @param provider
+ * @param hoprTokenAddress
+ * @param nativeAmount
+ * @param hoprAmount
+ * @param recipient
+ */
+export default async function main(
+  privateKey: string,
+  providerStr: string,
+  hoprTokenAddress: string,
+  nativeAmountStr: string,
+  hoprAmountStr: string,
+  recipient: string
+): Promise<void> {
+  log.normal("Funding via wallet", {
+    provider: providerStr,
+    hoprTokenAddress,
+    nativeAmount: nativeAmountStr,
+    hoprAmount: hoprAmountStr,
+    recipient,
+  });
+
+  const provider = new providers.JsonRpcProvider(providerStr);
+  const wallet = new Wallet(privateKey).connect(provider);
+
+  // check if wallet has enough balance to do transactions
+  const walletBalance = await wallet.getBalance();
+  if (walletBalance.lt(utils.parseEther("0.1"))) {
+    throw Error("Wallet balance is less than 0.1");
+  }
```

Comment:
should we also check if wallet has enough hopr balance?

Reply:
added